### PR TITLE
Style Guide and Diataxis Review - Part IV

### PR DESCRIPTION
**Pull request by charlesthompson3**
Applied style guide and diataxis changes to Chrysalis documentation including the:

- Mainnet
- Devnet
- Private Tangle
- Protocol RFCs
- Resources
- and FAQ sections

### CODE REVIEW

**Comment by charlesthompson3** [comment]
@iotaledger/tech-writers last Chrysalis one🙌

File: docs/devnet.md [hunk 1]
```diff
@@ -1,37 +1,40 @@
 ---
-description: The IOTA foundation provides load-balanced public devnet endpoints, with MQTT and the HTTP REST API enabled.
+description: The IOTA foundation provides load-balanced public devnet endpoints, where MQTT and the HTTP REST API enabled.
```
Comment (aleksei-korolev) [inline docs/devnet.md:2]: Something is missing here. API is/are enabled?

File: docs/faq.md [hunk 2]
```diff
@@ -41,18 +42,20 @@ The Chrysalis release still depends on the Coordinator/Compass to run and issue
 
 ## When is a Transaction on the Network Considered Final/Irreversible?
 
-On average, confirmation times on the new network are around 10 seconds. Once a transaction is set to confirm it’s final transaction, you don’t have block confirmations like with blockchain.
+On average, confirmation times on the new network are around 10 seconds. Once a transaction is set to confirm its final transaction, you do not have block confirmations like with blockchain.
```
Comment (aleksei-korolev) [inline docs/faq.md:45]: Tbh I'd check this one with the devs, I don't understand what it is supposed to mean.

File: docs/faq.md [hunk 3]
```diff
@@ -41,18 +42,20 @@ The Chrysalis release still depends on the Coordinator/Compass to run and issue
...
 -  To secure your spent addresses during the migration, [Firefly](https://github.com/iotaledger/firefly) will try to find a new bundle that reveals the least amount of additional private key parts compared to previous signatures.
-- This process will take 10 minutes per spent address and upon completion you will be presented with a risk calculation (very high, high, medium, low, very low). We recommended that you repeat the process if it returns a bundle with medium risk or higher, particularly for significant sums of IOTA. You have the option to select which addresses you want to mine for, and again which you want to rerun the process for.
+- This process will take 10 minutes per spent address and upon completion, you will be presented with a risk calculation (very high, high, medium, low, very low). We recommended that you repeat the process if it returns a bundle with medium risk or higher, particularly for significant sums of IOTA. You have the option to select which addresses you want to mine for, and again which you want to rerun the process for.
```
Comment (aleksei-korolev) [inline docs/faq.md:61]: I disagree. This comma disconnects "upon completion" from the rest of the clause, so now it is not immediately clear to which clause it belongs to, even if technically proper.

File: docs/mainnet.md [hunk 4]
```diff
@@ -1,18 +1,19 @@
 ---
-description: The IOTA foundation provides load-balanced public mainnet endpoints, with MQTT and the HTTP REST API enabled. 
+description: The IOTA foundation provides load-balanced public mainnet endpoints, where MQTT and the HTTP REST API enabled.
```
Comment (aleksei-korolev) [inline docs/mainnet.md:2]: same concern.

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: docs/devnet.md (hunk 1 above)
Comment (charlesthompson3) [reply docs/devnet.md:2]: "Are" 👍

File: docs/faq.md (hunk 2 above)
Comment (charlesthompson3) [reply docs/faq.md:45]: 👍

File: docs/faq.md (hunk 3 above)
Comment (charlesthompson3) [reply docs/faq.md:61]: Difference in styles🤷; wont fight this comma but you know there will be more.

File: docs/mainnet.md (hunk 4 above)
Comment (charlesthompson3) [reply docs/mainnet.md:2]: Same answer

**Comment by charlesthompson3** [comment]
@lucas-tortora one last time, I promise 🙏

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**